inst/include/Rcpp/api/meat/Rcpp_eval.h

```diff
@@ -54,7 +54,7 @@ inline SEXP Rcpp_eval(SEXP expr, SEXP env) {
     SET_TAG(CDDR(CDR(call)), ::Rf_install("interrupt"));
 
     // execute the call
-    Shield<SEXP> res(::Rf_eval(call, R_GlobalEnv));
+    Shield<SEXP> res(::Rf_eval(call, env));
```

Comment:
Actually this switch is not necessary, since we already have the `env` from `evalq`. I involve this when trying to simplify `Rcpp_eval`. I will update.